### PR TITLE
add swagger img

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Add Swagger logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ It embraces [JSON for Linked Data (JSON-LD)](http://json-ld.org) and [Hydra Core
 
 Build a working and fully-featured CRUD API in minutes. Leverage the awesome features of the tool to develop complex and
 high performance API-first projects.
-
+<img src="https://camo.githubusercontent.com/3a20ce067d38ecc857843b57b0d926e3c718f11f/687474703a2f2f323433347a6432396d697364336534613466316537336b692e7770656e67696e652e6e6574646e612d63646e2e636f6d2f77702d636f6e74656e742f7468656d65732f6b6576696e6c656172796e65742f6173736574732f696d672f737761676765722d6c6f676f4032782e706e67" alt="Swagger" data-canonical-src="http://2434zd29misd3e4a4f1e73ki.wpengine.netdna-cdn.com/wp-content/themes/kevinlearynet/assets/img/swagger-logo@2x.png" width="50" height="50">
```
Comment (dunglas) [inline README.md]: Please add a blank line before. Is this an official badge?

Comment (Simperfit) [inline README.md]: It is the official logo. Maybe we could add the .json in the repository and validate against the validator ?

**Comment by teohhanhui** [comment]
:-1:

It's not obvious to those who don't recognize the Swagger logo. There should at least be a link?

In any case, the Swagger spec seems to have been moved under the Open API Initiative and is now known as the [The OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification).

**Comment by Simperfit** [comment]
@teohhanhui I think we should add the swagger validator logo which means that our spec is valid, for that I don't know if we can do it in travis or if we have to save json file somewhere in order to test it.

Or we can add it and then rely on the swagger tester in travis ?
If the build passes, then the swagger spec is valid anyway.

WDYT ?

**Comment by teohhanhui** [comment]
There's a JSON schema here: https://github.com/OAI/OpenAPI-Specification/blob/931dc1bbed8b63e68ccb4130d97de02199d58b71/schemas/v2.0/schema.json

**Comment by Simperfit** [comment]
We have to dump the one we have with the console command then validate it against the schema validator to have the image.

**Comment by teohhanhui** [comment]
@Simperfit I really don't get what you mean by "validation image" (badge?), and why do we need that at all?

What we should try to add in our Behat test suite is validation of the generated Swagger spec (by API Platform) against the Swagger JSON Schema (linked above).

**Comment by Simperfit** [comment]
@teohhanhui Yes I mean the validation badge.

BTW we are already testing the validity of the schema in travis (swagger validate ..).

I just wanted to inform in the readme that we support Swagger and that the schema is valide according to travis ?

**Comment by teohhanhui** [comment]
The validation badge is meant for published Swagger spec of an API, not a library.

**Pull request closed without merging**